app/controllers/drafts_controller.rb

```diff
@@ -0,0 +1,49 @@
+class DraftsController < ApplicationController
+  requires_login
+
+  skip_before_action :check_xhr, :preload_json
+
+  def index
```

Comment:
We should add a request spec for this :)

Reply:
I added something for this in `draft_controller_spec.rb`, let me if it's insufficient @tgxworld.

Reply:
@tgxworld  thoughts?